client/dashboard/components/action-list/action-item.scss

```diff
@@ -0,0 +1,57 @@
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/variables";
+
+.action-item {
+	padding: $grid-unit-20 0;
+	border-bottom: 1px solid $gray-100;
+
+	&:first-child {
+		padding-top: $grid-unit-10;
+	}
+
+	&:last-child {
+		padding-bottom: $grid-unit-10;
+		border-bottom: none;
+	}
+
+	.action-item__title {
+		@include heading-medium;
+	}
+
+	.action-item__description {
+		@include body-medium;
+	}
```

Comment:
Where are these two mixins defined (heading-medium and body-medium)

Reply:
Yep, they're [here](https://github.com/WordPress/gutenberg/blob/e5c93c9d8d24bcd6b1b9e76b51557809afb62a67/packages/base-styles/_mixins.scss#L54).